pkgs/tools/security/vulnix/default.nix

```diff
@@ -1,16 +1,18 @@
-{ stdenv, pythonPackages, fetchurl, callPackage, nix }:
+{ stdenv, pythonPackages, fetchurl, callPackage, nix, ronn }:
 
 pythonPackages.buildPythonApplication rec {
   name = "${pname}-${version}";
```

Comment:
Please drop this line. `name` is set automatically by `buildPython*`.

Reply:
sorry, didn't know that